prover/Cargo.toml

```diff
@@ -5,9 +5,9 @@ members = [
     "prover_fri_types",
 
     # binaries
-    "prover",
+    # "prover",
     "circuit_synthesizer",
-    "setup_key_generator_and_server",
+    # "setup_key_generator_and_server",
```

Comment:
These are commented out because they do not compile and (AFAIK) are not used.

Reply:
ack. Then we should remove them in the next PR.